end2end/spring_boot_mysql.py

```diff
@@ -1,30 +1,24 @@
-from utils.test_safe_vs_unsafe_payloads import test_safe_vs_unsafe_payloads
-from spring_boot_mysql.test_two_sql_attacks import test_two_sql_attacks
-from spring_boot_mysql.test_ip_blocking import test_ip_blocking
-from spring_boot_mysql.test_bot_blocking import test_bot_blocking
-from spring_boot_mysql.test_ratelimiting import test_ratelimiting_per_user, test_ratelimiting
-from utils.EventHandler import EventHandler
+from __init__ import events
+from utils import App, Request
 
-payloads = {
-    "safe": { "name": "Bobby" },
-    "unsafe": { "name": 'Malicious Pet", "Gru from the Minions") -- ' }
-}
-urls = {
-    "disabled": "http://localhost:8083/api/pets/create",
-    "enabled": "http://localhost:8082/api/pets/create"
-}
+spring_boot_mysql_app = App(8082)
 
-event_handler = EventHandler()
-event_handler.reset()
-test_safe_vs_unsafe_payloads(payloads, urls, user_id="123") # Test MySQL driver
-test_safe_vs_unsafe_payloads(payloads, urls, "/mariadb", user_id="456") # Also test MariaDB driver
+spring_boot_mysql_app.add_payload(
+    "sql_mysql",test_event=events["spring_mysql_boot_mysql_attack"],
```

Comment:
Maybe nice to have separate files for these events? kind of what we did here: https://github.com/AikidoSec/firewall-php/tree/main/tests/server/test_allowed_ip

Reply:
would be for a possible next PR, right now the file isn't superbig and feel like it works to keep an overview